python/src/test/java/org/apache/zeppelin/python/IPythonInterpreterTest.java

```diff
@@ -434,9 +434,6 @@ public void run() {
           InterpreterResult result = interpreter.interpret("import time\ntime.sleep(1000)",
                   getInterpreterContext());
           waiter.assertEquals(InterpreterResult.Code.ERROR, result.code());
-          waiter.assertEquals(
-                  "IPython kernel is abnormally exited, please check your code and log.",
-                  result.message().get(0).getData());
```

Comment:
During my tests `result.message()` was always `null`. We can investigate this later, as this has nothing to do with the issue.
In the summary of https://github.com/apache/zeppelin/actions/runs/3939354095 you can also see several errors which have errors because of this `null`.

The `0` errors indicate the `null` error:
![grafik](https://user-images.githubusercontent.com/454320/213204320-9a4d9b08-7b6c-4100-8b32-b9bfd3a56d99.png)

After clicking on the message, the exception is displayed.
![grafik](https://user-images.githubusercontent.com/454320/213204661-dff85334-b36a-4d77-a59f-2f0d987c1f0f.png)

If you then look in the affected line, you will see a similar line.
https://github.com/apache/zeppelin/blob/0209d0d0d40e979bd284d89ec3eb6a90a34bf84a/spark/interpreter/src/test/java/org/apache/zeppelin/spark/SparkIRInterpreterTest.java#L110-L128

You should not make assertions within additional threads.  See also: https://rules.sonarsource.com/java/RSPEC-2186